packages.txt

```diff
@@ -28,7 +28,7 @@ gomplate@cloudposse
 goofys@cloudposse
 gosu@cloudposse
 groff
-helm@cloudposse==2.14.0-r0
+helm@cloudposse==2.14.1-r0
```

Comment:
```suggestion
helm@cloudposse
```

Reply:
(this is not directed at you @Nuru, just from a previous discussion on Slack)

lets *never* rely on pinning versions in `geodesic` to fix problems of individual customers.
let's pin versions for customers in their own `geodesic` environment.
otherwise we're just playing a constant game of fixing things in one place, while breaking it in another.
pinning only works if we're surgical about it; updating pinning in `geodesic` and then updating the customer to point to it is about as stable as pinning to `master`.
and that's not stable